gamification-github-webapp/src/main/resources/locale/navigation/group/platform/administrators_en.properties

```diff
@@ -1 +1 @@
-administration.gamificationGithubWebHook=GitHub Gamification
```

Comment:
if you delete a label, you will have to delete it in all other files, see : 
https://developer.exoplatform.org/#id-how-to-delete-a-translation-string